index.js

```diff
@@ -302,7 +300,7 @@ module.exports = (oldPlugin) => {
               message,
               triggered_by: triggeredBy,
               triggered_at: triggeredAt,
-              forwarded_by: (Array.isArray(forwardedBy) && forwardedBy[0]) || '',
+              forwarded_by: (Array.isArray(forwardedBy) && forwardedBy[0]) || this.oldPlugin.getAccount(),
```

Comment:
Disagree with this change. The weird `peer.` and other "plug-in" addresses are removed in LPIv2.

In LPIv2, only participants (sender/receiver/connector) have addresses, plugins do not. So I think using the deprecated plug-in address is more confusing than helpful.